Greg = 14
Marcia = 12 
Peter = 11
Jan = 10
Bobby = 8
Cindy = 6
Cousin Oliver = 8

Mean = 9.857142857
Median = 10
Mode = 8
Var = 6.408163265
Std = 2.531435021
Se = 0.956792504

In [1]:
import pandas as pd
import numpy as np
import statistics
import matplotlib.pyplot as plt
%matplotlib inline

df = pd.DataFrame()
df['age'] = [14, 12, 11, 10, 8, 6, 8]

df['age'].describe()

count     7.000000
mean      9.857143
std       2.734262
min       6.000000
25%       8.000000
50%      10.000000
75%      11.500000
max      14.000000
Name: age, dtype: float64

In [2]:
#statistics.mode(df['age'])

In [3]:
np.std(df['age'], ddof=0)

2.531435020952764

In [4]:
np.var(df['age'], ddof=0)

6.408163265306122

In [5]:
np.std(df['age'])/np.sqrt(len(df['age']))

0.95679250365151347

## *Question

For this activity, would I use Std, Var and SE for the population or the sample? I figured because I'm using data for all of the kids listed, I would be finding for the population. 

Using these estimates, if you had to choose only one estimate of central tendency and one estimate of variance to describe the data, which would you pick and why?

I would pick the mean at 9.9 with std at 2.5. While the median is close at 10, the mean at 9.9 does a slightly better job capturing the skew to the left because of the two lower ages at 8. The std does a good job capturing the narrow distribution of the data set. 

After Cindy turns 7, mean, std, var and se changes. But mode and median remains the same. 

After replacing Oliver with a 1-year old Jessica, I would use median at 10 as the central tendency because it doesn't significantly skew the data to the left. For the measure of variance, I would still use std because at 4.0 it shows how much wider the distribution got compared to the last dataset. 

In [6]:
df = pd.DataFrame()
df['ratings'] = [20, 23, 17, 5]

df['ratings'].describe()

count     4.000000
mean     16.250000
std       7.889867
min       5.000000
25%      14.000000
50%      18.500000
75%      20.750000
max      23.000000
Name: ratings, dtype: float64

I would say around 18.5% of adults were fans of the Brady Bunch. The SciPhi Fanatic subscribers are a niche group so their ratings wouldn't be an accurate portrayal of the general adult population's sentiments. Hence, their 5% result is an outlier. However, with broader groups like the 3 other subscription, we probably get a more reflective result of the population. Hence, the median, which is not heavily influenced by outliers is the better measure of central tendency here. 